### DISCUSSION TITLE
failed to send brpc batch, error=Host is down

### DISCUSSION THREAD

**Discussion by Rmuyu** [apache/doris#15007] [A - General / Q&A]
使用1.2.0版本，集群中每台be隔一段时间就会报错：
`W1212 14:02:38.698901 12112 fragment_mgr.cpp:251] Got error while opening fragment 9fe1cb45683e4e1f-ac354e4712d84d1b, query id: 9fe1cb45683e4e1f-ac354e4712d84d18: RPC error(error -1): failed to send brpc batch, error=Host is down, error_text=[E110]Fail to read from Socket{id=257 fd=243 addr=10.10.1.126:8060:51520} (0x0x7f8783bda340): Connection timed out [R1][E112]Not connected to 10.10.1.126:8060 yet, server_id=257 [R2][E112]Not connected to 10.10.1.126:8060 yet, server_id=257 [R3][E112]Not connected to 10.10.1.126:8060 yet, server_id=257 [R4][E112]Not connected to 10.10.1.126:8060 yet, server_id=257 [R5][E112]Not connected to 10.10.1.126:8060 yet, server_id=257 [R6][E112]Not connected to 10.10.1.126:8060 yet, server_id=257 [R7][E112]Not connected to 10.10.1.126:8060 yet, server_id=257 [R8][E112]Not connected to 10.10.1.126:8060 yet, server_id=257 [R9][E112]Not connected to 10.10.1.126:8060 yet, server_id=257 [R10][E112]Not connected to 10.10.1.126:8060 yet, server_id=257, client: 10.10.1.123, latency = 190070329/n    @     0x55bb1dffa0aa  doris::Status::ConstructErrorStatus()
    @     0x55bb220f5ed4  doris::vectorized::VDataStreamSender::Channel::_wait_last_brpc()
    @     0x55bb220ee19c  doris::vectorized::VDataStreamSender::Channel::close_wait()
    @     0x55bb220f3efd  doris::vectorized::VDataStreamSender::close()
    @     0x55bb1e91385a  doris::PlanFragmentExecutor::open_vectorized_internal()
    @     0x55bb1e914a25  doris::PlanFragmentExecutor::open()
    @     0x55bb1e8ecdbc  doris::FragmentExecState::execute()
    @     0x55bb1e8f01eb  doris::FragmentMgr::_exec_actual()
    @     0x55bb1e8f088a  _ZNSt17_Function_handlerIFvvEZN5doris11FragmentMgr18exec_plan_fragmentERKNS1_23TExecPlanFragmentParamsESt8functionIFvPNS1_20PlanFragmentExecutorEEEEUlvE_E9_M_invokeERKSt9_Any_data
    @     0x55bb1eb9b815  doris::ThreadPool::dispatch_thread()
    @     0x55bb1eb91bff  doris::Thread::supervise_thread()
    @     0x7f87f16aaea5  start_thread
    @     0x7f87f19bd9fd  __clone
    @              (nil)  (unknown)`
并且报错期间，服务查询一直超时未响应

**Comment by yuanyuan8983** [comment]
Add me WeChat: cyllyy810222, let's take a closer look at this problem.
